society-of-biblical-literature-fullnote-bibliography.csl

```diff
@@ -906,7 +971,26 @@
   <macro name="locators-journal">
     <choose>
       <if type="article-journal">
-        <text variable="page" prefix=": "/>
+        <!-- Add support for individually paginated, electronic journal articles per https://sblhs2.com/2018/05/03/electronic-journals-with-individually-paginated-articles/ -->
+        <choose>
+          <if variable="number">
+            <text term="article-journal" form="verb-short" prefix=": "/>
+            <text variable="number" prefix=" " suffix=", "/>
+            <choose>
+              <if variable="page">
+                <label variable="page" form="short"/>
+                <text variable="page"/>
+              </if>
+              <else>
+                <!-- Is there not a term for "no page"? See CMS17 §10.42 where the same abbreviation applies to "no place," "no publisher," and "no page." -->
+                <text value="n.p."/>
```

Comment:
you're sure SBL wants n.p.? And in what situations exactly?

Reply:
I believe so in this specific case when (in Zotero) the "number" variable has an entry to trigger the "individually published electronic article" subtype. SBLHS2 lists "no page" as one of the meanings for "n.p." in §8.1.3. A few examples:

- Richard D. Weis, “_Biblia Hebraica Quinta_ and the Making of Critical Editions of the Hebrew Bible,” _TC_ 7 (2002): art. 6, ¶¶45–48, http://jbtc.org/v07/Weis2002.html.
- Richard D. Weis, “_Biblia Hebraica Quinta_ and the Making of Critical Editions of the Hebrew Bible,” _TC_ 7 (2002): art. 6, n.p., http://jbtc.org/v07/Weis2002.html.
- Weis, Richard D. “_Biblia Hebraica Quinta_ and the Making of Critical Editions of the Hebrew Bible.” _TC_ 7 (2002): art. 6, n.p. http://jbtc.org/v07/Weis2002.html.

Reply:
Really? But there's also https://sblhs2.com/2016/08/09/hts-teologiese-studies/

Reply:
I think the two blog posts are a bit contradictory.

Reply:
@denismaier, that's what it looks like to me. SBLHS2 doesn't explicitly speak to the pilcrows, but those would come in per CMS17 §10.43.

That said, I've asked for clarification on the HTS post as follows

_Per [this guidance](https://sblhs2.com/2018/05/03/electronic-journals-with-individually-paginated-articles/), should the "Lombaard, Christo" bibliography entry read "art. 2851, pp. 1–5." instead of "art. 2851."?_

Was there something else you were seeing that indicated a possible contradiction between the two posts?